Doc/library/tarfile.rst

```diff
@@ -495,19 +497,23 @@ be finalized; only the internally used file object will be closed. See the
    The *filter* argument specifies how ``members`` are modified or rejected
    before extraction.
    See :ref:`tarfile-extraction-filter` for details.
-   It is recommended to set this explicitly depending on which *tar* features
-   you need to support.
+   It is recommended to set this explicitly only if unusual *tar* features
+   are required.
 
    .. warning::
 
-      Never extract archives from untrusted sources without prior inspection.
+      The default filter is set to ``filter='data'`` to prevent the most
+      dangerous security issues, read the :ref:`tarfile-extraction-filter`
+      section for details.
```

Comment:
```suggestion
      The default filter is set to ``filter='data'`` to prevent the most
      dangerous security issues. Read the :ref:`tarfile-extraction-filter`
      section for details.
```